src/components/Spotify.jsx

```diff
@@ -0,0 +1,10 @@
+// Create a new component called Spotify that returns allow me to embed a Spotify playlist
+import * as Spotify from 'react-spotify-embed';
+
+function SpotifyCustomPlayer() {
+  return (
+    <Spotify.Spotify link="https://open.spotify.com/playlist/1j6flBxNJC54Jb7AafwuGD?si=d65e948c8f904a35" />
```

Comment:
What do you think about adding the Spotify playlist id more dynamic? 
I'm thinking in two options.. We could receive the playlist from the url and, if the user didn't provide one, we can use something by default.
Or we can add the playlist id as a envar, to avoid committing in case that we need to change the playlist